lib/pacemaker/pcmk_injections.c

```diff
@@ -485,55 +494,55 @@ modify_configuration(pe_working_set_t * data_set, cib_t *cib, pcmk_injections_t
         CRM_ASSERT(rc == pcmk_ok);
     }
 
-    if (injections->watchdog) {
+    if (injections->watchdog != NULL) {
         rc = update_attr_delegate(cib, cib_sync_call | cib_scope_local,
-                             XML_CIB_TAG_CRMCONFIG, NULL, NULL, NULL, NULL,
-                             XML_ATTR_HAVE_WATCHDOG, injections->watchdog, FALSE, NULL, NULL);
-
+                                  XML_CIB_TAG_CRMCONFIG, NULL, NULL, NULL, NULL,
+                                  XML_ATTR_HAVE_WATCHDOG, injections->watchdog,
+                                  FALSE, NULL, NULL);
         CRM_ASSERT(rc == pcmk_ok);
     }
 
-    for (gIter = injections->node_up; gIter != NULL; gIter = gIter->next) {
-        char *node = (char *)gIter->data;
+    for (iter = injections->node_up; iter != NULL; iter = iter->next) {
+        char *node = (char *) iter->data;
 
         out->message(out, "inject-modify-node", "Online", node);
 
         cib_node = pcmk__inject_node_state_change(cib, node, true);
         CRM_ASSERT(cib_node != NULL);
 
         rc = cib->cmds->modify(cib, XML_CIB_TAG_STATUS, cib_node,
-                                      cib_sync_call | cib_scope_local);
+                               cib_sync_call|cib_scope_local);
         CRM_ASSERT(rc == pcmk_ok);
         free_xml(cib_node);
     }
 
-    for (gIter = injections->node_down; gIter != NULL; gIter = gIter->next) {
+    for (iter = injections->node_down; iter != NULL; iter = iter->next) {
         char xpath[STATUS_PATH_MAX];
-        char *node = (char *)gIter->data;
+        char *node = (char *) iter->data;
 
         out->message(out, "inject-modify-node", "Offline", node);
 
         cib_node = pcmk__inject_node_state_change(cib, node, false);
         CRM_ASSERT(cib_node != NULL);
 
         rc = cib->cmds->modify(cib, XML_CIB_TAG_STATUS, cib_node,
-                                      cib_sync_call | cib_scope_local);
+                               cib_sync_call | cib_scope_local);
```

Comment:
There's a bunch more of these in this commit.  I don't think they all get removed by a later commit, so it'd still be worth doing.

Reply:
added commit on top